test/typescript-karma/Dashboards/Component/tests.js

```diff
@@ -76,7 +76,8 @@ test('Board without data connectors and HighchartsComponent update', async funct
                 type: 'Highcharts',
                 chartOptions: {
                     title: {
-                        text: void 0
+                        text: void 0,
+                        style: {}
```

Comment:
Why this one is required?

Reply:
Because of Highcharts changes. Right now the style param is required in the TS type.

We discussed it with @bre1470, and adding style is the simplest way of solving problem. However in the future we should consider using optional `style` in HC, like it was in the past.